manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BlenderFoundation.Blender
+PackageVersion: 4.0.1
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- 3ds
+- fbx
+- glb
+- gltf
+- obj
+- ply
+- stl
+ProductCode: '{B557A37F-811D-4A98-9306-FB34E069D999}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.blender.org/download/release/Blender4.0/blender-4.0.1-windows-x64.msi

```

Comment:
```suggestion
  InstallerUrl:  https://ftp.halifax.rwth-aachen.de/blender/release/Blender4.0/blender-4.0.1-windows-x64.msi
```

Reply:
Thats a specific mirror, which won’t be the best URL for all users. Its best to point to a CDN wherever possible, just like previous versions of this package do